featurebyte/session/bigquery.py

```diff
@@ -154,6 +155,12 @@ def initializer(self) -> BaseSchemaInitializer:
     def is_threadsafe(cls) -> bool:
         return True
 
+    def get_additional_execute_query_kwargs(self) -> dict[str, Any]:
+        job_config = bigquery.QueryJobConfig(
+            default_dataset=f"{self.database_name}.{self.schema_name}"
```

Comment:
Do we need to quote the identifiers?

Reply:
I did that initially, but there will be an error complaining that the project name / dataset name have invalid characters if we quote the identifiers here. It behaves correctly without quotes.

Reply:
I guess no need since the identifier names are quite restrictive